### ISSUE TITLE
`new SQLBuilder('MySQL')` Fails on Windows

### ISSUE THREAD

**Issue by em-hshibuki** [planetarydev/json-sql-builder2#2]
The following code is fails on Windows.
```
const SQLBuilder = require('json-sql-builder2');
const sql = new SQLBuilder('MySQL');
```

The details of error is:
```
Debugger listening on ws://127.0.0.1:36065/9492f253-abbe-409b-b76d-178e0321aa1b
Debugger attached.
path.js:28
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at Object.join (path.js:489:7)
    at SQLBuilder._register (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:485:33)
    at _.forEach (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:459:10)
    at arrayEach (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\lodash\lodash.js:516:11)
    at Function.forEach (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\lodash\lodash.js:9342:14)
    at SQLBuilder._loadModules (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:457:5)
    at new SQLBuilder (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:48:8)
    at Object.<anonymous> (c:\Users\hidori\GitHub\node-json-sql-builder2-test\index.js:2:13)
    at Module._compile (module.js:649:14)
Waiting for the debugger to disconnect...
TypeError: Path must be a string. Received undefined
path.js:28
    at assertPath (path.js:28:11)
    at Object.join (path.js:489:7)
    at SQLBuilder._register (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:485:33)
    at _.forEach (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:459:10)
    at arrayEach (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\lodash\lodash.js:516:11)
    at Function.forEach (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\lodash\lodash.js:9342:14)
    at SQLBuilder._loadModules (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:457:5)
    at new SQLBuilder (c:\Users\hidori\GitHub\node-json-sql-builder2-test\node_modules\json-sql-builder2\lib\builder.js:48:8)
    at Object.<anonymous> (c:\Users\hidori\GitHub\node-json-sql-builder2-test\index.js:2:13)
    at Module._compile (module.js:649:14)
```

So, builder.js contains following codes:
```
		this.currentModulePath = path.join('../sql/', file.split('/sql')[1]);
```
```
		op.sqlPath = './' + this.currentModulePath.split('/sql/')[1];
```
these are NOT GOOD for Windows.

On Windows, the file path is separated by `\`. not `/`.

**Comment by planetarydev** [comment]
Oh yes, you are absolutely wright! This is a bug.
I have changed every split-command from `file.split('/sql'))` to `file.split(path.sep + 'sql')` and all code-blocks where I build manually a file-path.

I could not test it on windows, because I've got no windows. The tests after changing the code on linux will run fine and everything works as expected.

Could you please do the Test on windows for me and run `npm test` in the console. I bump the version and published at npm with version 1.0.17

Thanks in advance and I will be glad to hear any result on your tests.

**Comment by hidori** [comment]
>Could you please do the Test on windows for me and run npm test in the console. I bump the version >and published at npm with version 1.0.17

The result is:
```
  Summary:

    146 Tests passing
```
YES! It's LGTM :-)

Thank you for making good product!

**Comment by planetarydev** [comment]
Thanks for testing. Awesome!
If you are working with MySQL and you will create or extend some Operators and Helpers please feel free to fork and make a pull-request with the new stuff.
